lms_crawling.py

```diff
@@ -0,0 +1,35 @@
+from selenium import webdriver
+import getpass
+
+driver = webdriver.Chrome('C:\chromedriver.exe')
+driver.get('https://ieilms.jbnu.ac.kr/login.php')
+
+id = input("학번을 입력하세요 : ")
+pw = getpass.getpass("비밀번호를 입력하세요 : ")
+driver.find_element_by_name('username').send_keys(id)
+driver.find_element_by_name('password').send_keys(pw)
+driver.find_element_by_xpath('//*[@id="region-main"]/div/div/div/div[1]/div[1]/div[2]/form/div[2]/input').click()
+courseList = driver.find_element_by_class_name('course_lists').find_elements_by_tag_name('li')
+courseName = []
+courseURL = []
+for i in courseList:
+    courseName.append(i.find_element_by_class_name('course-title').text.split(' ')[0])
+    courseURL.append(i.find_element_by_tag_name('a').get_attribute('href'))
+
+for i in range(len(courseList)):

```

Comment:
```suggestion
for i in range(len(courseList)):
    if courseName[i] == '[혁신교육개발원]':
        continue
```
1학년 bsm과목을 듣는 경우에는 혁신교육개발원에서 제공하는 강의가 있어서 오류가 발생합니당.
아예 제외시켜주는 코드를 넣음 좋을 것 같아요!

Reply:
헐랭구리 그런 문제가 있었군요?!! 오키오키 수정해서 올릴게용!